### ISSUE TITLE
repeating tasks on MacOS Catalina

### ISSUE THREAD

**Issue by gijshvg** [mhoeher/opentodolist#29]
I downloaded and installed the latest .dmg for MacOS. I use Catalina, Macbook Pro. My other installs are Android and linux (ubuntu 18) app. I like OpenTodoList very much, on Android it is by far the most flexible tool.
App installed without problems, syncing is fine. Making new items also.
But I cannot make repeating tasks on MAcOS, like on Android or Linux. 
Do I overlook something, is it just not possible on Apple machines, or what?
GijsH

**Comment by mhoeher** [comment]
Hi @gijshvg, thanks for reaching out!

Nice to hear you like the app so far. Regarding setting the recurrence on macOS... this is basically another reminder for me to migrate to another icon provider... :disappointed:

To provide some details:

Currently, the app uses FontAwesome for the icons. This project delivers not one but several font files. The tricky part is, that some icons are available e.g. only in the *regular* font style while others are only on the *bold* one, for example. And it seems this is what happens here again... on the particular tool button, the wrong font is set. Some font rendering engines (obviously the ones on Android and also Linux) then seem to try to take the icon form another font (i.e. the "sibling" font, where it is in fact available). However, on macOS the icon cannot be resolved and hence is not shown.

Long story short: You can - until a fix is released - still set recurrence: The icon is right between the due date selection icon and the "edit" icon for notes:

![image](https://user-images.githubusercontent.com/1911135/99935287-aac3c880-2d60-11eb-981a-a54bb473a37a.png)

So if you click on the right spot into the blank area, you still should get the dialog to set a recurrence. To fix the issue, I created another ticket over on [GitLab](https://gitlab.com/rpdev/opentodolist/-/issues/386) (where the app  is developed). As soon as a first test version is available, I'll let you know here so you can give it a try.

**Comment by gijshvg** [comment]
Hi @mhoeher: Thank you for your adequate response. Indeed, I can find the
 icon now you have told me where to click :|) 
I will wait for te new
 version, but this helps
 already.

----------------------------------------

From: Martin Höher
 <notifications@github.com>
Sent: 11/23/20 7:55 AM
To:
 mhoeher/opentodolist <opentodolist@noreply.github.com>
Cc: gijshvg
 <gijshvg@gmail.com>, Mention <mention@noreply.github.com>
Subject: Re:
 [mhoeher/opentodolist] repeating tasks on MacOS Catalina (#29)

Hi
 @gijshvg, thanks for reaching out!

Nice to hear you like the app so
 far. Regarding setting the recurrence on macOS... this is basically
 another reminder for me to migrate to another icon provider... 😞

To
 provide some details:

Currently, the app uses FontAwesome for the
 icons. This project delivers not one but several font files. The tricky
 part is, that some icons are available e.g. only in the regular font style
 while others are only on the bold one, for example. And it seems this is
 what happens here again... on the particular tool button, the wrong font
 is set. Some font rendering engines (obviously the ones on Android and
 also Linux) then seem to try to take the icon form another font (i.e. the
 "sibling" font, where it is in fact available). However, on macOS the icon
 cannot be resolved and hence is not shown.

Long story short: You can -
 until a fix is released - still set recurrence: The icon is right between
 the due date selection icon and the "edit" icon for
 notes:

[image]

So if you click on the right spot into the blank
 area, you still should get the dialog to set a recurrence. To fix the
 issue, I created another ticket over on GitLab (where the app is
 developed). As soon as a first test version is available, I'll let you
 know here so you can give it a try.

—
You are receiving this because
 you were mentioned.
Reply to this email directly, view it on GitHub, or
 unsubscribe.

**Comment by mhoeher** [comment]
Hi @gijshvg, I implemented a minor fix. If you like, you could download a fresh build from here: https://gitlab.com/rpdev/opentodolist/-/jobs/867955598/artifacts/file/build-macos/app/OpenTodoList.dmg

If you do, would be great if you could let me know if the icon is now visible for you :wink: Thanks!

**Comment by mhoeher** [comment]
Tested the change and merged it meanwhile to the master branch. The fix will be part of the next release.
